### DISCUSSION TITLE
ImportError: cannot import name 'ImageRegistrator' from 'kornia.geometry'

### DISCUSSION THREAD

**Discussion by MehdiShekarnabi** [kornia/kornia#1290] [Potential Issue] [answered]
### Describe the bug

Hello, 
After installing with "pip install kornia" I got the version 0.5.10
Trying the simple "from kornia.geometry import ImageRegistrator" I get the following error:
ImportError: cannot import name 'ImageRegistrator' from 'kornia.geometry'
Can you advise please?

### Reproduction steps

```bash
1. pip install kornia
2. from kornia.geometry import ImageRegistrator
...
```

### Expected behavior

expected not to get an ImportError

### Environment

```shell
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```
- PyTorch Version (e.g., 1.0): 1.9.0
- OS (e.g., Linux):
- How you installed PyTorch (`conda`, `pip`, source):
- Build command you used (if compiling from source):
- Python version: 3.8.6
- CUDA/cuDNN version: cu111
- GPU models and configuration: 
- Any other relevant information:
```


### Additional context

_No response_

**Comment by edgarriba** [comment, marked as answer]
@MehdiShekarnabi `ImageRegistrator` it's not in the latest pypi release -- only in main branch for now and under testing before our next release. To try, install kornia from source as suggested [here](https://github.com/kornia/kornia#from-source-using-pip)
`pip install git+https://github.com/kornia/kornia`
